app/src/main/res/values/design_time_attributes.xml

```diff
@@ -16,6 +16,10 @@
         tellus eu rhoncus pellentesque. Quisque et leo faucibus, aliquam nisi ut,
         feugiat lectus.
     </string>
+
+    <string name="design_time_sampletext">
```

Comment:
Can you use the lorem ipsum text instead? Feels a bit weird to me to have the text of one specific podcast baked into the app. We don't even know if we are allowed to use that text because of their license

Reply:
Sure, i will